releasenotes/notes/munir-add-with-otel-prefix-covagent-f4650caf7d2fcab7.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Ensures host metrics and a subset of kafka metrics submitted via OTLP are prefixed with `otel.`. This avoids conflicts with Datadog Agent metrics.
```

Comment:
We don't need a changelog since this only impacts converged agent, and converged agent is not yet released.